tst/EnergyPlus/unit/Timer.unit.cc

```diff
@@ -56,12 +56,20 @@
 
 // Standard library headers
 #include <chrono>
+#include <cstdlib>
 #include <thread>
 
 using namespace EnergyPlus;
 
-TEST_F(EnergyPlusFixture, DISABLED_Timer_ticktock)
+TEST_F(EnergyPlusFixture, Timer_ticktock)
 {
+    // This test fails on GitHub Actions CI and I am not sure why.
+    // I still want it exercised regularly, so I am going to let CI
+    // just wrap it up early and pass.  This test has spent most of its
+    // life DISABLE_d anyway, so this is a net improvement.
+    if (std::getenv("CI")) {
+        return;
+    }
```

Comment:
Same thing as numThreads here.  There's something super odd about the CI image that causes this to fail.  I do not understand.  I'm happy to let it run on everyone's personal machines regularly and not worry about this on CI.